examples/postgresql-typeorm/README.md

```diff
@@ -0,0 +1,45 @@
+# Postgresql example
+
+A simple user app using tinyhttp and [PostgreSQL](https://www.postgresql.org/).
+
+## Setup
+
+### Install dependencies
+
+```sh
+pnpm install
+```
+
+### Install PostgreSQL
+
+[PostgreSQL Wiki](https://wiki.postgresql.org/wiki/Detailed_installation_guides) has a good list of guides for installing postgres on different operating systems.
+
+### Create a database
+
+Use [createdb](https://www.postgresql.org/docs/10/app-createdb.html#:~:text=createdb%20creates%20a%20new%20PostgreSQL,the%20SQL%20command%20CREATE%20DATABASE.) to create a database
+
+```
+createdb [dbname]
+```
+
+### Configure TypeORM
+
+Setup database settings inside `ormconfig.json` file
```

Comment:
You forgot to put an example of these settings. It's unclear what settings to set. But there's a file in this same repo so it would make sense to write that settings are located in some file blah blah or just copy paste them here